core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -110,6 +110,13 @@ public Icon getIconByClassSpec(Object iconClassSpec) {
         return getIconByClassSpec(iconClassSpec.toString());
     }
 
+    /**
+     * @return the amount of icons.
+     */
+    public int size() {
```

Comment:
I don’t see a reason for this to be in the public API, to me the test doesn’t add value, you could check a particular icon is in there maybe?

Reply:
You are right, I just wanted to check the coreIcons size, so I removed that again, since it is already accessible.